pkg/notify/elasticsearch.go

```diff
@@ -23,6 +23,11 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws/signer/v4"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/sha1sum/aws_signing_client"
+	"time"
```

Comment:
built-in imports should go in the first group. Please move "time" into first group (with "fmt")